templates/v3/includes/_field_badge_select.html

```
@@ -0,0 +1,170 @@
+    choice either - there is no clear control on the JS side. {% endcomment %}
+    {% if not selected %}<option value="">{{ placeholder|default:"Select a badge" }}</option>{% endif %}
+    {% for option in options %}
+      <option value="{{ option.value }}"
+              {% if not option.selectable %}disabled{% endif %}
+              {% if option.value and option.value|stringformat:"s" == selected|stringformat:"s" %}selected{% endif %}>
+        {{ option.rank }} {{ option.name }} — {{ option.detail }}
+      </option>
+    {% endfor %}
+  </select>
+  <div class="dropdown badge-select" :class="{ 'dropdown--open': open, 'dropdown--has-value': selected }" x-cloak>
+    <div class="dropdown__trigger"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)